aspnetcore/fundamentals/servers/kestrel/when-to-use-a-reverse-proxy.md

```diff
@@ -0,0 +1,39 @@
+---
+title: When to use a reverse proxy with the ASP.NET Core Kestrel web server
+author: rick-anderson
+description: Learn about when to use a reverse proxy in front of Kestrel, the cross-platform web server for ASP.NET Core.
+monikerRange: '>= aspnetcore-5.0'
+ms.author: riande
+ms.custom: mvc
+ms.date: 05/04/2020
+no-loc: [appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
+uid: fundamentals/servers/kestrel/when-to-use-a-reverse-proxy
+---
+
+# When to use Kestrel with a reverse proxy
```

Comment:
Cross link to https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/proxy-load-balancer?view=aspnetcore-5.0